src/components/views/settings/tabs/user/EncryptionUserSettingsTab.tsx

```diff
@@ -33,19 +33,49 @@ import { ResetIdentityPanel } from "../../encryption/ResetIdentityPanel";
  *                        This happens when the user doesn't have a key a recovery key and the user clicks on "Set up recovery key" button of the RecoveryPanel.
  *  - "reset_identity": The panel to show when the user is resetting their identity.
  */
-type State = "loading" | "main" | "set_up_encryption" | "change_recovery_key" | "set_recovery_key" | "reset_identity";
+export type State =
+    | "loading"
+    | "main"
+    | "set_up_encryption"
+    | "change_recovery_key"
+    | "set_recovery_key"
+    | "reset_identity_compromised"
+    | "reset_identity_forgot";
 
-export function EncryptionUserSettingsTab(): JSX.Element {
-    const [state, setState] = useState<State>("loading");
-    const setUpEncryptionRequired = useSetUpEncryptionRequired(setState);
+interface Props {
```

Comment:
Since we have multiple components in the file:

```suggestion
interface EncryptionUserSettingsTabProps {
```